lib/stop.js

```diff
@@ -0,0 +1,5 @@
+(function() {
+  'use strict';
+
+  console.log('stopping');
+})();
```

Comment:
It does nothing now except log to console. I wanted it there as placeholder, though, once it is actually a service managed by `forever`. Then it will stop the service gracefully.